go/vt/vtadmin/cluster/cluster.go

```diff
@@ -1264,13 +1265,12 @@ func (c *Cluster) getTablets(ctx context.Context) ([]*vtadminpb.Tablet, error) {
 
 		go func(tablet *topodatapb.Tablet) {
 			defer wg.Done()
-
 			var state vtadminpb.Tablet_ServingState
 			_, err := c.Vtctld.RunHealthCheck(ctx, &vtctldatapb.RunHealthCheckRequest{
 				TabletAlias: tablet.Alias,
 			})
 			if err != nil {
```

Comment:
just dropping a placeholder so i don't forget about the `UNKNOWN` vs `NOT_SERVING` distinction here